src/plugins/wizard/public/application/utils/wizard_state_validation.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Ajv from 'ajv';
+import wizardStateSchema from './schema.json';
+
+export const validateWizardState = (wizardState) => {
+  const ajv = new Ajv();
+  const validateState = ajv.compile(wizardStateSchema);
```

Comment:
Since the schema does not change, can we move the compile to be outside this function so that it isnt run during each function call?

Reply:
+1

Reply:
Fixed